files/nginx.conf

```diff
@@ -18,8 +18,19 @@ http {
   # this is necessary for us to be able to disable request buffering in all cases
   proxy_http_version 1.1;
 
+  lua_shared_dict token_dict 5m;
+
   # will run before forking out nginx worker processes
-  init_by_lua_block { require "cjson" }
+  init_by_lua_block {
+    require "cjson"
+
+    local token_file = io.open('/usr/local/openresty/nginx/token.txt', 'r')
+    if token_file then
```

Comment:
Should we log an error in the else case?

Reply:
Yes something like should do the trick
```
    if token_file then
        local data = token_file:read()
        ngx.shared.token_dict:set("ecr_token", data)
        token_file:close()
    else
        ngx.log(ngx.ERR, "Failed to open token file: /usr/local/openresty/nginx/token.txt")
    end
```
Will push a fix the maintainer is responding.
Thanks for the reviews.